vmdk_plugin/plugin.go

```diff
@@ -209,6 +221,10 @@ func (d *vmdkDriver) unmountVolume(name string) error {
 // Name and driver specific options passed through to the ESX host
 func (d *vmdkDriver) Create(r volume.Request) volume.Response {
 
+	// Lock this volume for create
+	d.lockVolume(r.Name)
+	defer d.unlockVolume(r.Name)
```

Comment:
Agreed with the remove part.
For create i think its necessary to protect the create protocol that has several not protected steps within the driver/plugin (create/attach/mkfs/detach)

what do you think?